website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -44,6 +44,7 @@ if (boolean(process.env.DEBUG_LOGIN) || process.env.NODE_ENV === "development")
         const user = {
           id: credentials.username,
           name: credentials.username,
+          role: "admin",
```

Comment:
This is temporal. I will refactor the debug login to allow select roles when login in later PR.